### ISSUE TITLE
On Windows, child processes are always spawned 'detached'

### ISSUE THREAD

**Issue by skleeschulte** [foreverjs/forever-monitor#147]
Is there a reason that on Windows child processes are always spawned 'detached'?
See here: [monitor.js, line 246-248](https://github.com/foreverjs/forever-monitor/blob/f6291b4f09d3de815dbbc70bf9e26768ad1b8d46/lib/forever-monitor/monitor.js#L246-L248)

I want to quit the child process (tree) when my script with forever-monitor exits.

**Comment by jac0117** [comment]
Right, why spawn it as detached? I have a script that will launch multiple APIs, but when I Ctrl+C on the terminal, the child processes stay alive.

**Comment by campersau** [comment]
These lines got added in https://github.com/foreverjs/forever-monitor/pull/62
